### PR TITLE
app/version: v1.0-dev

### PR DESCRIPTION
**Pull request by gsora**
Instead of going with version v0.20, use v1.0.

category: misc
ticket: none

### CODE REVIEW

**Review by pinebit — APPROVED** [review]

**Review by KaloyanTanev — APPROVED** [review]

File: app/peerinfo/peerinfo_test.go [hunk 1]
```diff
@@ -23,22 +23,28 @@ func TestPeerInfo(t *testing.T) {
 	now := time.Now()
 	const gitCommit = "1234567"
 
+	// when a major release happens, charon will only be compatible with a single version
+	versions := version.Supported()
+	if len(versions) < 2 {
+		versions = append(versions, versions...)
+	}
```
Comment (KaloyanTanev) [inline app/peerinfo/peerinfo_test.go:30]: If there is a single version supported we... duplicate it?

Comment (gsora) [reply app/peerinfo/peerinfo_test.go:30]: yes, as a corner case in this test, in order to keep the test working when there will be more than one supported

Comment (KaloyanTanev) [reply app/peerinfo/peerinfo_test.go:30]: Not the prettiest, but fine by me, beauty is in the eye of the beholder 😄
